tests/unit/helpers.js

```diff
@@ -615,8 +615,11 @@ export function JedSpy(data = {}) {
 /*
  * Creates a stand-in for a jed instance,
  */
-export function fakeI18n({ lang = config.get('defaultLang') } = {}) {
-  return makeI18n({}, lang, JedSpy);
+export function fakeI18n({
+  lang = config.get('defaultLang'),
+  includeJedSpy = true,
+} = {}) {
+  return makeI18n({}, lang, includeJedSpy ? JedSpy : undefined);
```

Comment:
AFAIK, there is no test for this but let me know if one should be added..

Reply:
> AFAIK, there is no test for this but let me know if one should be added..

No worries. Storybook will break if we accidentally change it.